mandatory/4-tax.js

```diff
@@ -17,7 +19,10 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let Price = calculateSalesTax(price);
```

Comment:
This looks good, but you might think about the name of this variable `Price`. Having 2 variables with very similar names (`price` and `Price`) might be confusing for other developers 😄

Reply:
thanks thats evdn confusing for me too

Reply:
i tired to give another name before, it wasn’t working then after few tries i put price to make it working